src/metrics/list_repos.rs

```diff
@@ -95,26 +111,18 @@ async fn count_pull_requests_graphql(
     )
     .unwrap();
 
-    // the reason you'll see `last:1` in the query below is b/c we don't actually need to iterate through every single PR
-    // to get the count of PRs created in the last X days. The GQL `search` query returns a type with the relevant `count` irregardless.
-    // The reason this query is so much faster to count PRs is b/c it's just making the one request and not iterating through any data
-    let query_string = format!(
-        r#"query {{
-            search(
-                query:"repo:{org_name}/{repo_name} is:pr created:>{date_str}",
-                type:ISSUE,
-                last:1,
-            ) {{
-                issueCount
-            }}
-        }}"#,
+    let query_string: String = format!(
+        r#"repo:{org_name}/{repo_name} is:pr created:>{date_str}"#,
         org_name = org_name,
         repo_name = repo_name,
         date_str = date_str,
     );
-
+    let q = QuerySearch::build_query(query_search::Variables {
+        query_string: query_string,
+    });
     let octo = octocrab::instance();
-    let response: Response<Data> = octo.graphql(&query_string).await?;
-
-    response.data.search.issue_count as usize
+    let response: Response<query_search::ResponseData> = octo.graphql_with_params(&q).await?;
```

Comment:
I was imagining we might make this

```suggestion
    let response: Response<query_search::ResponseData> = octo.graphql_with_params::<QuerySearch>(query_search::Variables { query_String }).await?;
```

and have the extension function do the work of invoking `QuerySearch::build_query`.

Reply:
An alternative might be something like

```rust
QuerySearch::build(...).execute(&octo)
```

In which case, we would implement the "extension trait" for [`QueryBody<Q>`](https://docs.rs/graphql_client/0.9.0/graphql_client/struct.QueryBody.html).

That might be more ergonomic and better?